addons/web/static/tests/views/fields/boolean_field.test.js

```diff
@@ -0,0 +1,245 @@
+import { expect, test } from "@odoo/hoot";
+import { check, click, press, queryAll, queryOne, uncheck } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import {
+    clickSave,
+    defineModels,
+    fields,
+    models,
+    mountView,
+    onRpc,
+} from "@web/../tests/web_test_helpers";
+
+class Partner extends models.Model {
+    bar = fields.Boolean({ default: true });
+
+    _records = [
+        { id: 1, bar: true },
+        { id: 2, bar: true },
+        { id: 3, bar: true },
+        { id: 4, bar: true },
+        { id: 5, bar: false },
+    ];
+}
+
+defineModels([Partner]);
+
+test("boolean field in form view", async () => {
+    await mountView({
+        resModel: "partner",
+        resId: 1,
+        type: "form",
+        arch: `
+            <form>
+                <label for="bar" string="Awesome checkbox"/>
+                <field name="bar"/>
+            </form>
+        `,
+    });
+    expect(`.o_field_boolean input`).toBeChecked({
+        message: "checkbox should still be checked",
+    });
+    expect(`.o_field_boolean input`).toBeEnabled({
+        message: "checkbox should not be disabled",
+    });
+
+    // uncheck the checkbox
```

Comment:
You can remove those comments. Utils from hoot are explicit enough (save for `// save`)